### DISCUSSION TITLE
Bug: Search in 'Members and permissions' leads to crash

### DISCUSSION THREAD

**Discussion by lt-bha** [GitbookIO/community#382] [0. General bugs and questions] [answered]
If I enter a something in the search box above the members list GitBook crashes and shows a 'Something went wrong page'.

**Comment by valentin0h** [comment]
Thanks for reporting @lt-bha 

We are on it and a fix is coming very soon.

**Comment by aikoii** [comment, marked as answer]
Hey @lt-bha! You might already know, but this has been fixed 🎉
